src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (a) => {
+    count++;
+
+    return (b) => {
```

Comment:
```suggestion
  return (a) => {
    return (b) => {
      count++;
```
Better to put this `count` inside the inner function